### ISSUE TITLE
Explore RAMSES validation and loading methods

### ISSUE THREAD

**Issue by razekmh** [SAFEHR-data/ramses-package#13]
**TLDR**: understand how to use the validation tools in RAMSES to help in validating OMOP data later

**Background**:
Ramses offers some validation tools to conform electronic health records to the standards of RAMSES data model. We would like to gain better understanding of the methods to help in planning and building a module that validate OMOP data and load it to the RAMSES database 

**Tasks**
- [x] read and follow the [Validate and load electronic health records](https://ramses-antibiotics.web.app/articles/load-data.html).
- [ ] summaries the general findings to document our understanding of the validate and load processes
